src/app/loadout/plug-drawer/SelectablePlug.tsx

```diff
@@ -28,13 +32,44 @@ export default function SelectablePlug({
   onPlugSelected(plug: PluggableInventoryItemDefinition): void;
   onPlugRemoved(plug: PluggableInventoryItemDefinition): void;
 }) {
-  const defs = useD2Definitions()!;
-
   const handleClick = useCallback(() => {
     selectable && onPlugSelected(plug);
   }, [onPlugSelected, plug, selectable]);
 
   const onClose = useCallback(() => onPlugRemoved(plug), [onPlugRemoved, plug]);
+
+  // Memoize the meat of the component as it doesn't need to re-render every time
+  const plugDetails = useMemo(
+    () => <SelectablePlugDetails plug={plug} displayedStatHashes={displayedStatHashes} />,
+    [plug, displayedStatHashes]
+  );
```

Comment:
I'm not sure, but I think this should help greatly reduce the amount of reconciler work when selecting mods.